src/Runner.Worker/JobRunner.cs

```diff
@@ -84,7 +84,14 @@ public async Task<TaskResult> RunAsync(AgentJobRequestMessage message, Cancellat
                 Trace.Info($"Creating job server with URL: {jobServerUrl}");
                 // jobServerQueue is the throttling reporter.
                 _jobServerQueue = HostContext.GetService<IJobServerQueue>();
-                VssConnection jobConnection = VssUtil.CreateConnection(jobServerUrl, jobServerCredential, new DelegatingHandler[] { new ThrottlingReportHandler(_jobServerQueue) });
+                var delegatingHandlers = new List<DelegatingHandler>() { new ThrottlingReportHandler(_jobServerQueue) };
+                message.Variables.TryGetValue("system.github.enable_http_redirects", out VariableValue enableHttpRedirects);
```

Comment:
i think we normally just use the feature flag name as the variable in other case.
ex: `DistributedTask.EnhancedAnnotations` both on the server and the runner, so it's easy to search and cleanup later.